.circleci/config.yml

```diff
@@ -18,7 +19,7 @@ jobs:
                 default: '4'
             mysql_version:
                 type: string
-                default: '5.7.22'
+                default: '8.0.33'
```

Comment:
Are we upgrading MySQL on the deployed instances as well?

Reply:
Yep, good catch. Thanks!